src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/BailNotificationGeneratorConfiguration.java

```diff
@@ -25,49 +25,49 @@ public class BailNotificationGeneratorConfiguration {
 
     @Bean("submitApplicationNotificationGenerator")
     public List<BailNotificationGenerator> submitApplicationNotificationGenerator(
-        HearingCentreSubmitApplicationPersonalisation hearingCentreSubmitApplicationPersonalisation,
-        LegalRepresentativeBailApplicationSubmittedPersonalisation legalRepresentativeBailApplicationSubmittedPersonalisation,
-        HomeOfficeBailApplicationSubmittedPersonalisation homeOfficeBailApplicationSubmittedPersonalisation,
-        ApplicantBailApplicationSubmittedPersonalisationSms applicantBailApplicationSubmittedPersonalisationSms,
-        BailGovNotifyNotificationSender notificationSender,
-        BailNotificationIdAppender notificationIdAppender) {
+            HearingCentreSubmitApplicationPersonalisation hearingCentreSubmitApplicationPersonalisation,
```

Comment:
Can you check if your IDE has the correct checkstyle sheet? Or may be the previous change was not matching with the checkstyle.

Reply:
I'm not too sure what happened with the commit which indented all these. It looks like as long as it is consistent the checkstyle passes. I've removed out the extra indentation in the latest commit.